kotlin/internal/jvm/compile.bzl

```diff
@@ -51,6 +51,11 @@ load(
 def _java_info(target):
     return target[JavaInfo] if JavaInfo in target else None
 
+def _is_android_sandwich(current_label_name, target):
+    """Checks if the given target is <target>_base target from android sandwich macro"""
+    is_base_target = current_label_name.removesuffix("_kt") == target.label.name.removesuffix("_base")
```

Comment:
Doesn't work on Bazel versions older than 5.1.0, https://github.com/bazelbuild/bazel/pull/14899

Reply:
@Nikolas-LFDesigns  thanks, I removed this impl and just switched the order in `deps` attribute. That also worked.